openwisp_controller/config/base/device.py

```diff
@@ -127,6 +127,9 @@ def __str__(self):
     def _has_config(self):
         return hasattr(self, 'config')
 
+    def _has_group(self):
+        return hasattr(self, 'config')
```

Comment:
Are you sure this is correct?

Shouldn't it be:

```suggestion
        return hasattr(self, 'group')
```